hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaTargetFilterQueryManagement.java

```diff
@@ -196,6 +196,12 @@ public long count() {
         return targetFilterQueryRepository.findById(targetFilterQueryId).map(tfq -> (TargetFilterQuery) tfq);
     }
 
+    @Override
+    public TargetFilterQuery getById(final Long targetFilterQueryId) {
+        return get(targetFilterQueryId).map(tfq -> (TargetFilterQuery) tfq)
```

Comment:
unnecessary cast, remove the whole `map` block

Reply:
done